config.json

```diff
@@ -1603,9 +1603,7 @@
 
         ],
         "prerequisites": [
-          "strings",
-          "conditionals",
-          "enumeration"
+
```

Comment:
As @iHiD commented, let's leave the prerequisites in.  The student has a way to get around having to do the exercises for the prerequisites, and it hopefully will avoid folks trying to refactor without having a good (enough) understanding of the prerequisites.

Reply:
Done!